### PR TITLE
Qbits use bestla as 3rd lib in cmake

### PR DESCRIPTION
**Pull request by zhewang1-intc**
## Type of Change

feature or bug fix or documentation or others
API changed or not

## Description

detail description 
JIRA ticket: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Review by a32543254 — APPROVED** [review]
LGTM

File: intel_extension_for_transformers/llm/operator/csrc/dispatcher/neural_speed.cmake [hunk 1]
```diff
@@ -0,0 +1,9 @@
+set(NEURAL_SPEED_URL https://github.com/intel/neural-speed.git)
+set(NEURAL_SPEED_TAG 18720b319d6921c28e59cc9e003e50cee9a85fcc) # kernel-only release v0.2
```
Comment (airMeng) [inline intel_extension_for_transformers/llm/operator/csrc/dispatcher/neural_speed.cmake:2]: @luoyu-intel I think a tag is more reasonable?

**Review by PenghuiCheng — APPROVED** [review]

File: intel_extension_for_transformers/llm/operator/csrc/dispatcher/neural_speed.cmake (hunk 1 above)
Comment (airMeng) [reply intel_extension_for_transformers/llm/operator/csrc/dispatcher/neural_speed.cmake:2]:
```suggestion
set(NEURAL_SPEED_TAG bestlav0.1)
```

Comment (airMeng) [reply intel_extension_for_transformers/llm/operator/csrc/dispatcher/neural_speed.cmake:2]: https://github.com/intel/neural-speed/releases/tag/bestlav0.1

Comment (zhewang1-intc) [reply intel_extension_for_transformers/llm/operator/csrc/dispatcher/neural_speed.cmake:2]: make sence

**Review by airMeng — APPROVED** [review]

**Pull request merged**